test/test.js

```diff
@@ -337,34 +336,38 @@ describe('Utilities', function() {
     });
   });
 
-  describe('#getValueCaseInsensitive_()', function() {
-    var getValueCaseInsensitive_ = OAuth2.getValueCaseInsensitive_;
-
-    it('should find identical keys', function() {
-      assert.isTrue(getValueCaseInsensitive_({'a': true}, 'a'));
-      assert.isTrue(getValueCaseInsensitive_({'A': true}, 'A'));
-      assert.isTrue(getValueCaseInsensitive_({'Ab': true}, 'Ab'));
-    });
-
-    it('should find matching keys of different cases', function() {
-      assert.isTrue(getValueCaseInsensitive_({'a': true}, 'A'));
-      assert.isTrue(getValueCaseInsensitive_({'A': true}, 'a'));
-      assert.isTrue(getValueCaseInsensitive_({'Ab': true}, 'aB'));
-      assert.isTrue(getValueCaseInsensitive_({'a2': true}, 'A2'));
+  describe('#witLowerCaseKeys_()', function() {
+    var witLowerCaseKeys_ = OAuth2.witLowerCaseKeys_;
+    var data = {
+      'a': true,
+      'A': true,
+      'B': true,
+      'Cc': true,
+      'D2': true,
+      'E!@#': true
+    };
+    var lowerCaseData = witLowerCaseKeys_(data);
+
+    it('should contain lower-case keys', function() {
```

Comment:
Note, a stronger test would compare the exact input/output of values for the function rather than the presence and lack of presence of certain keys.

i.e. Combine these 2 tests. 'a' and 'A' could be one test like:

```js
assert.deepEqual(lowerCaseData, {
  'a': true,
  'b': true,
  'cc': true,
  'd2': true,
  'e!@#': true,
});
```

This will assert everything else not in the object `isUndefined` too. You could also then inline `lowerCaseData`.